components/status_dropdown/status_dropdown.tsx

```diff
@@ -212,7 +217,6 @@ export default class StatusDropdown extends React.PureComponent {
         return (
             <MenuWrapper
                 onToggle={this.onToggle}
-                style={this.props.style}
```

Comment:
To confirm, does someone else pass this now?

Reply:
Hi @lieut-data , I remove it because I didnt find componenet MenuWrapper  use props "style".
In component "components/sidebar/sidebar_header/sidebar_header.tsx", component SidebarHeader call component StatusDropdown without sending any props.

Reply:
Yeah it doesn't look like we use this anywhere. Definitely removeable.